erp/export/mappers.py

```diff
@@ -0,0 +1,115 @@
+from dataclasses import fields
+from typing import List, Tuple
+
+from erp import schema
+from erp.export.models import ETALAB_SCHEMA_FIELDS, EtalabModel
+from erp.models import Erp, Accessibilite
+
+
+def map_value_from_schema(schema_enum, data):
+    return schema_enum[[y[0] for y in schema_enum].index(data)][1]
+
+
+def map_list_from_schema(schema_enum, data):
+    if not len(data):
+        return None
+
+    result = set()
+    for d in data:
+        choice = schema_enum[[y[0] for y in schema_enum].index(d)][1]
+        result.add(choice)
+
+    return result
+
+
+def map_erp_to_json_schema(erps: List[Erp]) -> Tuple[List[str], List[EtalabModel]]:
+    headers = ETALAB_SCHEMA_FIELDS
+
+    results = []
+    for erp in erps:
+        try:
+            erp.accessibilite
+        except Accessibilite.DoesNotExist:
+            continue
+
+        o = EtalabModel(
```

Comment:
Possibily silly idea: how about exposing a `to_etalab_json` method in the `Erp` model? so we could write:

```python
results = [erp.to_etalab_json() for erp in erps]
```

Reply:
Not sure about, this, it would be kind of a leak of infra in the model. I'd rather return a a predefined dict. Or maybe be more OOP and inject a presenter.